Getting-Started/Backoffice/Login/index.md

```diff
@@ -35,7 +35,7 @@ The "Forgot password?" link allows your backoffice users to reset their password
       </mailSettings>
     </system.net>
 
-This feature can be turned off completely using the `allowPasswordReset` configuration, see: [/Documentation/Reference/Config/umbracoSettings/#security](/Documentation/Reference/Config/umbracoSettings/#security) 
+This feature can be turned off completely using the `allowPasswordReset` configuration, see: [UbracoSettings - Security](/Reference/Config/umbracoSettings/index.md/#security) 
```

Comment:
```suggestion
This feature can be turned off completely using the `allowPasswordReset` configuration, see: [UmbracoSettings - Security](../../../Reference/Config/umbracoSettings/#security) 
```
A few small errors in the relative link, also index.md get's stripped from the URL so no reason to have it there 😉

Reply:
@jmayntzhusen I don't see why index.md gets stripped from the URL, whenever I open the link without index.md, it directs to UmbracoDocs/Reference/Config/umbracoSettings/ and not Security section in index.md

Reply:
If you click this link: https://our.umbraco.com/documentation/Reference/Config/umbracoSettings/#security it should work 🙂 and that is the link my suggested change would send you to